runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java

```diff
@@ -161,11 +161,12 @@ protected boolean discoverAddressSpace(final UUID streamId,
                     String message = String.format("getStreamAddressMap[{%s}] stream has been " +
                                     "trimmed at address %s and this space is not covered by the " +
                                     "loaded checkpoint with start address %s, while accessing the " +
-                                    "stream at version %s", this, streamAddressSpace.getTrimMark(),
+                                    "stream at version %s. Looking for a new checkpoint.", this,
+                            streamAddressSpace.getTrimMark(),
                             getCurrentContext().checkpointSuccessStartAddr, maxGlobal);
-                    log.warn(message);
+                    log.info(message);
                     if (options.ignoreTrimmed) {
-                        log.warn("getStreamAddressMap[{}]: Ignoring trimmed exception for address[{}].",
+                        log.debug("getStreamAddressMap[{}]: Ignoring trimmed exception for address[{}].",
```

Comment:
Did you figure out why we were seeing the huge number at the end of the message ?
2019-06-27T03:26:49.390Z WARN CorfuDB-DCN-Publisher-0 AddressMapStreamView - getStreamAddressMap[{ae2a@448367}] stream has been trimmed at address 407912 and this space is not covered by the loaded checkpoint with start address -1, while accessing the stream at version 9223372036854775807

Reply:
Yes, that is when we do remaining() which goes up to Long.MAX